src/test/functions/fetchCoins.test.js

```diff
@@ -0,0 +1,9 @@
+import fetchCoins from '../../services/fetchCoins';
+
+describe('Fetch Coins', () => {
+  test('gets the correct data', async () => {
+    const data = await fetchCoins();
+    expect(data).toBeInstanceOf(Array);
+    expect(data[0].id).not.toBeNull();
+  });
+});
```

Comment:
As part of the project requirements, kindly ensure that your app has unit test for several components of the app and some pure functions in redux code. :wink: